model/src/main/java/com/basistech/rosette/dm/AttributeKey.java

```diff
@@ -24,13 +24,13 @@
  */
 enum AttributeKey {
     BASE_NOUN_PHRASE("baseNounPhrases"),
+    CATEGORIZER_RESULTS("categorizerResults"),
+    DEPENDENCY("dependency"),
+    EMBEDDING("embeddings"),
```

Comment:
Ok.  I guess I'm paranoid from the old C days when people used enums as ints, and code depended on order.  Impossible from java enums, I think.  Binary compat?  It must just work for java...